test/integration/features/cmd-profile.feature

```diff
@@ -186,5 +186,3 @@ Feature: Profile commands
     | delete    |
     | logs      |
     | stop      |
-    | version   |
```

Comment:
Failing since now `minishift version --profile foo` doesn't exit with 1. See https://ci.centos.org/job/minishift-pr/4077/consoleFull.

Also, it now make sense, since version is not related to profile and shouldn't exit with 1.

Reply:
@budhrg ```minishift version --profile foo``` returns the version, Does not matter whatever parameters are passed after version string. Now my question is why ```--profile foo``` does not report then like ```minishift ip --profile foo``` ?
```
$ ./minishift version --profile foo
minishift v1.16.1+e3f98ab0

$ ./minishift ip --profile foo
Profile 'foo' doesn't exist, Use 'minishift profile set foo' or 'minishift start --profile foo' to create
```

Reply:
`minishift ip` gives you the ip address of the VM, which means it needs an existing VM, that's why the error msg..

but `minishift version` should run any way.. people might want to check the version before they start using it.

Reply:
> Now my question is why --profile foo does not report then like minishift ip --profile foo does?

Reply:
It is important to understand why it doesn't report... as this needs to be logically grouped.
I am doubtful about the placing within the source code.

Reply:
> but minishift version should run any way.. people might want to check the version before they start using it.

@anjannath Then how do you justify this ```minishift status --profile foo```. I think some other logic is working behind this
```
$ ./minishift status --profile foo
Profile 'foo' doesn't exist, Use 'minishift profile set foo' or 'minishift start --profile foo' to create
```
@budhrg I am still not convinced. Don't you think this is a regression because IMO the current pr changes breaking the scenario in cmd-profile.feature file ?

Reply:
This output i am getting on minishift master
```
$ ./minishift version --profile xyz
Profile 'xyz' doesn't exist, Use 'minishift profile set xyz' or 'minishift start --profile xyz' to create
```
This output i am getting on this pr
```
$ ./minishift version --profile pqr
minishift v1.16.1+e3f98ab0
```
@anjannath please argue if you think it was a issue before and this pr fixes it ?

Reply:
output as from PR is expected... ignoring `--profile` (or any input)...

`m version --foo bar` should always work...